scripts/waifulib/xcompile.py

```diff
@@ -754,6 +799,28 @@ def configure(conf):
 
 		conf.env.PSP_RENDER_TYPE = psp.render_type
 		conf.env.PSP_BUILD_PRX = psp.build_prx
+	elif conf.options.IOS_OPTS:
+		values = conf.options.IOS_OPTS.split(',')
+		if len(values) != 1:
+			conf.fatal('Invalid --ios paramater value!')
+
+		simulatorvalues = ['true', 'false']
+		if values[0] not in simulatorvalues:
+			conf.fatal("Invalid build for simulator answer, you must answer with true or false!")
+		conf.ios = ios = iOS(conf, values[0])
+		conf.env['cshlib_PATTERN'] = 'lib%s.dylib'
+		conf.env['cxxshlib_PATTERN'] = 'lib%s.dylib'
+		conf.env['FRAMEWORK_ST'] = ['-framework']
+		conf.env['FRAMEWORKPATH_ST'] = '-F%s'
+		conf.env['FRAMEWORK'] = [ 'Foundation', 'UIKit', 'QuartzCore', 'GameController', 'CoreMotion', 'SystemConfiguration', 'CoreFoundation', 'CFNetwork', 'AVFoundation', 'CoreGraphics' ]
+
+		conf.environ['CC'] = ios.cc()
+		conf.environ['CXX'] = ios.cxx()
+		
+		conf.env.CFLAGS += ios.cflags()
+		conf.env.CXXFLAGS += ios.cflags()
+		conf.env.LINKFLAGS += ios.linkflags()
+		conf.env.DEST_OS2 = 'ios'
```

Comment:
Again, check if you can set `DEST_OS`. Maybe similarly to what we do on Android, which by default gets detected as `linux`